README.md

```diff
@@ -24,14 +24,15 @@ To depend on `swift-nio-http2`, put the following in the `dependencies` of your
 
     .package(url: "https://github.com/apple/swift-nio-http2.git", from: "1.19.2"),
 
-The most recent versions of SwiftNIO HTTP/2 support Swift 5.5.2 and newer. The minimum Swift version supported for SwiftNIO HTTP/2 releases are detailed below:
+The most recent versions of SwiftNIO HTTP/2 support Swift 5.6 and newer. The minimum Swift version supported for SwiftNIO HTTP/2 releases are detailed below:
 
 SwiftNIO HTTP/2     | Minimum Swift Version
 --------------------|----------------------
 `1.0.0 ..< 1.18.0`  | 5.0
 `1.18.0 ..< 1.21.0` | 5.2
 `1.21.0 ..< 1.23.0` | 5.4
-`1.24.0 ...`        | 5.5.2
+`1.24.0 ..< 1.26.0` | 5.5.2
```

Comment:
Before we drop 5.5 support we'll need to tag a final release supporting 5.5. As there are unreleased API additions it will be released as `1.26.0` so this range will be `1.24.0 ..< 1.27.0`

Reply:
Sounds good. Was expecting to hold fire until I knew whether the next release was major or minor, but now you've told me I've updated the draft PR to expect a minor release, so expecting a second to drop support for 5.5. See 53db1e8.